core: introduce ceph-exporter (#11317)
rook/rook: Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

## Prometheus Instances

> With the Prometheus operator running, we can create service monitors that will watch the Rook cluster, which will now have two sources for metrics collection

Reviewer: nit: new paragraph
```suggestion
With the Prometheus operator running, we can create service monitors that will watch the Rook cluster.

There are two sources for metrics collection:
```

The following Grafana dashboards are available:

> - [Ceph - Cluster](https://grafana.com/grafana/dashboards/2842)

Reviewer: Did your editor change these? In Rook docs we are mostly using the `*` for bullet points for consistency

Author: Yes, Mrkdown lint extension changed it I think.
